apps/opentelemetry_semantic_conventions/lib/container_attributes.ex

```diff
@@ -0,0 +1,214 @@
+defmodule OpenTelemetry.SemanticConventions.ContainerAttributes do
+  # This is an auto-generated file
+  @moduledoc """
+  OpenTelemetry Semantic Conventions for Container attributes.
+  """
+
+  @doc """
+  The command used to run the container (i.e. the command name).
+
+  ### Notes
+
+  If using embedded credentials or sensitive data, it is recommended to remove them to prevent potential leakage.
+
+
+  ### Example
+      iex> OpenTelemetry.SemanticConventions.ContainerAttributes.container_command()
+      :"container.command"
+  """
+  @spec container_command :: :"container.command"
+  def container_command do
+    :"container.command"
+  end
+
+  @doc """
+  All the command arguments (including the command/executable itself) run by the container. [2]
+
+
+
+  ### Example
+      iex> OpenTelemetry.SemanticConventions.ContainerAttributes.container_commandargs()
+      :"container.command_args"
+  """
+  @spec container_commandargs :: :"container.command_args"
+  def container_commandargs do
+    :"container.command_args"
+  end
+
+  @doc """
+  The full command run by the container as a single string representing the full command. [2]
+
+
+
+  ### Example
+      iex> OpenTelemetry.SemanticConventions.ContainerAttributes.container_commandline()
+      :"container.command_line"
+  """
+  @spec container_commandline :: :"container.command_line"
+  def container_commandline do
+    :"container.command_line"
+  end
+
+  @typedoc """
+  The CPU state for this data point.
+
+  ### Options
+  * `:user` ^[e](`m:OpenTelemetry.SemanticConventions#experimental`)^ - When tasks of the cgroup are in user mode (Linux). When all container processes are in user mode (Windows).
+  * `:system` ^[e](`m:OpenTelemetry.SemanticConventions#experimental`)^ - When CPU is used by the system (host OS)
+  * `:kernel` ^[e](`m:OpenTelemetry.SemanticConventions#experimental`)^ - When tasks of the cgroup are in kernel mode (Linux). When all container processes are in kernel mode (Windows).
+
+  """
+  @type container_cpu_state() :: :user | :system | :kernel | atom()
+
+  @doc """
+  The CPU state for this data point.
+
+
+  ### Example
+      iex> OpenTelemetry.SemanticConventions.ContainerAttributes.container_cpu_state(:user)
+      :user
+      
+      iex> OpenTelemetry.SemanticConventions.ContainerAttributes.container_cpu_state(:custom_value)
+      :custom_value
+  """
+  @spec container_cpu_state(container_cpu_state()) :: :user | :system | :kernel | atom()
+  def container_cpu_state(option) do
+    :"container.cpu.state"
+
+    case option do
```

Comment:
```suggestion
  def container_cpu_state(option) do
    case option do
```

Are these atoms floating around intentional?

Reply:
Template typo